src/components/TodoItem/TodoItem.tsx

```diff
@@ -0,0 +1,60 @@
+import { useState } from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoItem: React.FC<Props> = ({ todo }) => {
+  const [edited] = useState(false);
```

Comment:
You set the state 'edited' to false and never change it. That makes the edit form in your JSX code unreachable. If you want to allow users to edit the todo, you should create a function that will change 'edited' state when necessary.